tools/triage/add_triage_signature.py

```diff
@@ -18,6 +18,7 @@
 from collections import OrderedDict, defaultdict, Counter
 from datetime import datetime
 from textwrap import dedent
+from tools.jira_client import JiraClientFactory
```

Comment:
the convention we (mostly) follow for imports is splitting to:
1. built-ins
2. third-party libraries
3. internal code

So this should be right beside
```
from tools.triage.common import get_cluster_logs_base_url
```

(this applies here and elsewhere)

Reply:
One day we'll have this file to pass pylint :sweat_smile:

Reply:
lot's of places still have this. can you have another look?

Reply:
:thinking:  it might be me but I don't see any out of order? Are you referring to this, right?

Reply:
adding some comments for those places (not for all though)

Reply:
Oh, thanks for pointing out, I was focusing only on the order of our packages :facepalm: 

Added `isort` so this can be done by a machine - which is much better than me at this job :grin:

Reply:
Yes, that's a great idea
Thank you!